pkg/util/mountOpts.go

```diff
@@ -148,6 +148,11 @@ func ProcessOptions(options []string, isTmpfs bool, sourcePath string) ([]string
 			foundNoSwap = true
 			newOptions = append(newOptions, opt)
 			continue
+		case "preserve_symlinks":
```

Comment:
In touch, chcon, chown, chmod, cp, cpio,dir, du, file, find, getfattr, ...

 this is called --no-dereference and -h.

       -h, --no-dereference
              affect each symbolic link instead of any referenced file (useful only on systems that can change the timestamps of a symlink)

Reply:
Great thinking!  Renamed it to `no_dereference`